test-gh-pages.sh

```diff
@@ -49,7 +49,7 @@ if [[ -d doc ]]; then
   fi
   cd $jekyll_root
   ruby scripts/create_global_toc.rb
-  jekyll serve
+  jekyll serve --host=0.0.0.0
```

Comment:
It's better to add here $@ and run the scripts as
./test-gh-pages.sh --host=0.0.0.0
It also gives ability to pass custom port
./test-gh-pages.sh --host=0.0.0.0 --port=4900